.github/workflows/end-to-end.yml

```diff
@@ -74,6 +74,17 @@ jobs:
           workspace: .maestro
           include-tags: privacyTest
 
+      - name: Security Tests
+        if: always()
+        uses: mobile-dev-inc/action-maestro-cloud@v1.4.1
+        with:
+          api-key: ${{ secrets.MOBILE_DEV_API_KEY }}
+          name: ${{ github.sha }}
+          app-file: apk/release.apk
+          android-api-level: 30
+          workspace: .maestro
+          include-tags: securityTest
+
```

Comment:
I ran the end-to-end test and it fails: https://github.com/duckduckgo/Android/actions/runs/7106164156/job/19344978847

<img width="634" alt="Screenshot 2023-12-05 at 20 50 29" src="https://github.com/duckduckgo/Android/assets/3471025/76fb366b-50c1-4fdb-b8b3-32be7daeb35e">
<br/><br/>
I tried running it locally and I had the same issue, this pop-up doesn't allow "run" to be clicked:
<img width="606" alt="Screenshot 2023-12-05 at 20 46 22" src="https://github.com/duckduckgo/Android/assets/3471025/62388a65-a7e8-480a-a805-c8cc8bac16c5">